etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiOrderConverter.java

```diff
@@ -152,13 +145,12 @@ private MessageHeader createMessageHeader() {
 
         messageHeader.setEvent(OML_CODING);
 
-        messageHeader.setMeta(
-                new Meta()
-                        .addTag(
-                                new Coding(
-                                        "http://terminology.hl7.org/CodeSystem/v2-0103",
-                                        "P",
-                                        "Production")));
+        var meta = new Meta();
+
+        // Adding processing id of 'P'
+        meta.addTag("http://terminology.hl7.org/CodeSystem/v2-0103", "P", "Production");
```

Comment:
Did we decide to persist the value that RS sends to us? Do we need to add this tag?

Reply:
We need to revisit this and see discuss it further. I think is a good opportunity to bring it up in our engineering block.